server/src/commandLine.mjs

```diff
@@ -89,6 +90,11 @@ const developerToolName = parsed.developerToolName || config.app.developerToolNa
 const proxy = parsed.proxy;
 const conduit = parsed.conduit;
 
+// Overriding multiUserConnections, if a configurartion is passed via CLI on staring MF server
+// Else default configuration 'warn' is used
+const multiUserConnections = parsed.multiUserConnections;
+config.multiUserConnections = multiUserConnections ? multiUserConnections : config.multiUserConnections
```

Comment:
Let's also validate the config here as well.

If the user provided config is not "allow", "warn" or "deny" we default to what's in the configuration file (warn). Those comparisons should be caps insensitive.